web-app/src/constants/index.ts

```diff
@@ -2,3 +2,7 @@ export const KEYBOARD_INPUT_DEBOUNCE_TIME = 300; // milliseconds
 
 export const MASUMI_URL = "https://masumi.network";
 export const KODOSUMI_URL = "https://kodosumi.com";
+export const SOKOSUMI_URL = "https://sokosumi.com";
+
+export const PASSWORD_MIN_LENGTH = 8;
+export const PASSWORD_MAX_LENGTH = 128;
```

Comment:
This is the default setting according to [better-auth](https://www.better-auth.com/docs/authentication/email-password#configuration), but I prefer to set it explicitly for zod-object validation.